firelyserver/releasenotes/releasenotes.rst

```diff
@@ -17,6 +17,64 @@ Upgrading Firely Server
+   #. In both cases you may also run the script manually beforehand. 
+   #. As always: make sure you have a backup of your database that is tested for restore as well.
+
+DevOps
+^^^^^^
+
+.. attention::
+
+   Because of a change in the devops pipeline there is no longer a ``Firely.Server.exe`` (formerly ``Vonk.Server.exe``) in the distribution zip file. You can run the server as always with ``dotnet ./Firely.Server.dll``
+
+Features
+^^^^^^^^
+
+#. Inferno, The ONC test tool: Firely Server now passes all the tests in this suite! With version 4.1.0 we specifically added features to pass the 'Multi-patient API' tests.
+
+   #. To test this for yourself we have set up a special instance of Firely Server together with Identity Server, working with a SQL Server repository and with Bulk Data Export enabled. You can access it at https://sql.server.fire.ly. 
+   #. ?? Mention the Postman collection with Inferno required data (https://www.getpostman.com/collections/766d29f0721556997e37)?
+
+#. Terminology support has been revamped. Previously you needed to choose between using the terminology services internal to Firely Server *or* external terminology services like from OntoServer or Loinc. With this version you can use both, and based on the codesystem or valueset involved the preferred terminology service is selected and queried. 
+
+   #. This works for terminology operations like ``$validate-code`` and ``$lookup``
+   #. It also works for validation, both explicitly with ``$validate`` and implicitly, when validating resources sent to Firely Server. 
+   #. The CodeSystem, ValueSet and ConceptMap resources involved are conformance resources and therefore always retrieved from the Administration database.
+   #. Responses may differ on details from previous versions of Firely Server, but still conform to the specification.
+   #. See :ref:`feature_terminology` for further details.
+
+#. ``$everything``: We now support the :ref:`feature_patienteverything` operation for single Patients. (For multiple patients, there is the Bulk Data Export feature.)
+#. Performance of $everything, Bulk Data Export and authorization on compartments improved. We added a special index to the database that keeps track which resource belongs to which compartment. First in SQL Server, MongoDB has less need for it. 
+#. SMART on FHIR: Support for token revocation. Reference tokens can be revoked, and Firely Server can check for the revocation.
+
+Fixes
+^^^^^
+
+#. SMART on FHIR: We have found ourselves that the authorization restrictions were bypassed when using _include or _revinclude in a FHIR Search. We solved this security issue immediately. 
```

Comment:
We should add a remark about the implications of this issue. Also we should place it more prominently in the release notes.

Reply:
I like the clear message at the top of the release notes. Next to that I'll also send out an email to our customers in case they don't read the release notes.